st2common/st2common/util/param.py

```diff
@@ -150,8 +152,32 @@ def _render(node, render_context):
     Render the node depending on its type
     '''
     if 'template' in node:
+        complex_type = False
+
+        LOG.debug("Rendering param of type: %s", type(node['template']))
+
+        if isinstance(node['template'], list) or isinstance(node['template'], dict):
+            node['template'] = json.dumps(node['template'])
```

Comment:
Yeah this is getting fed by our supported types (Unless I'm mistaken here) which are all supported in JSON.